tensorflow/lite/kernels/reverse_test.cc

```diff
@@ -77,6 +77,48 @@ TEST(ReverseOpTest, FloatMultiDimensions) {
                         17, 18, 15, 16, 13, 14, 23, 24, 21, 22, 19, 20}));
 }
 
+TEST(ReverseOpTest, FloatAxes) {
```

Comment:
nit: axis

Reply:
@renjie-liu , thanks for the comments, i have updated as suggested by you.Kindly check and approve.

Regards
Amit